src/parser/preprocessor.c

```diff
@@ -1126,6 +1127,10 @@ static void start_function_macro_expansion(const macro_call_t *call)
 			parameter->not_specified = true;
 		}
 	}
+    if (current_call.macro->is_pragma){
```

Comment:
wrong indentation again... maybe you should get a better editor

Reply:
sorry i didn't pay attention,
i use vim, it's just that i'm not used to contributing to projects, i should stick to project specific settings next times